hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/AbstractMultiFileWriter.java

```diff
@@ -110,7 +110,7 @@ public void init(StoreScanner sourceScanner, WriterFactory factory) {
     return paths;
   }
 
-  protected abstract Collection<StoreFileWriter> writers();
+  public abstract Collection<StoreFileWriter> writers();
```

Comment:
nit: should have Javadoc on a public method.

Reply:
Thanks for pointing it out.